portfolio/src/main/webapp/index.html

```diff
@@ -9,10 +9,34 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Hello! My name is Eashan.</p>
+      <p>I'm a CS major at the Georgia Institute of Technology and I have a passion for social good. In an constantly technologizing society, there is only so much we can do to help others.
+          I aspire to use my technical experience to make a difference on the world. Coming out of high school, I actively looked for ways to help out others. Now, it's my goal to do something
+          impactful with my skills.</p>
+      <p>Here is a link to my linkedin: https://www.linkedin.com/in/eashanbhatt</p>
+      <p>Before I get carried away with the rest of this page, I just wanted to highlight one of the organizations I'm in that has played a big role on my passion for cs.</p>
+      <p>Bits of Good is a nonprofit organization that works with local nonprofits to create products that improve their efficiency and impact on the community. I have always loved getting involved
+          with the community in any way I could and Bits of Good helps by combining that with technical abilities. One of the projects I worked on was involving the creation of an online
+          tutoring hub that links college students with students of low-income stature that may not be able to afford tutors and get the best education they can because of financial struggles.
+          I have since transitioned to working on the operations of the club so that I can get experience in a different area while still helping others. The community within Bits of Good has
+          also helped me to want to learn more about different technologies and is my motivation for trying unique stylings that are shown on this page.</p>
+      <p>This website presents a random collections of things that I like. Also, I'm practicing my web development skills and learning new things with CSS :)</p>
+      <p>In my free time, I play a lot of video games, especially since I have been in quarantine. Some of the games I play/games I enjoy playing include:</p>
+      <ul id="game-list">
+          <li>Apex Legends</li>
+          <li>Call of Duty</li>
+          <li>Fortnite</li>
+          <li>Minecraft</li>
+          <li>Town of Salem</li>
+      </ul>
+      <p>Click here to get a random iconic quote from Spongebob:</p>
+      <button onclick="addRandomGreeting()">*cue Spongebob transition*</button>
       <div id="greeting-container"></div>
+      <br/>
+      <p>Here are some great quotes</p>
+      <ul id="quote-list">
```

Comment:
since this is empty, save it for a future PR

Reply:
Fixed